openmc/mgxs_library.py

```diff
@@ -856,9 +857,35 @@ def set_scatter_mgxs(self, scatter, nuclide='total', xs_type='macro'):
         check_value('domain_type', scatter.domain_type,
                     ['universe', 'cell', 'material'])
 
+        # Methods of representing anisotropic scattering besides
+        # Legendre expansions have not been implemented yet in openmc.mgxs.
+        # Therefore check to make sure the XsData has been set to
+        # legendre scattering.
+        if (self.scatt_type != 'legendre'):
+            msg = 'Anisotrpic scattering representations other than ' \
+                  'Legendre expansions have not yet been implemented in ' \
+                  'openmc.mgxs.'
+            raise ValueError(msg)
+
+        # If the user has not defined XsData.order, then we will set
+        # the order based on the data within scatter.
+        # Otherwise, we will check to see that XsData.order to match
+        # the order of scatter
+        if self.order is None:
+            self.order = scatter.legendre_order
+        else:
+            check_value('legendre_order', scatter.legendre_order,
+                        [self.order])
```

Comment:
Yeah, I verified.  first line of code in check_value:
```if value not in accepted_values:``` will throw a syntax error since `in` operates on a list.